rails_docker/docker-compose.yml

```diff
@@ -9,6 +9,12 @@ services:
     ports:
       - "5432:5432"
 
+  redis:
```

Comment:
@Yutna I think we don't need this for Production, normally for Production we will use another Service for Redis like https://elements.heroku.com/addons/redistogo

Reply:
Let's leave it by default so that it can be removed when needed